packages/instant/src/components/install_wallet_panel_content.tsx

```diff
@@ -1,19 +1,15 @@
 import * as React from 'react';
```

Comment:
Since this is no longer generic, we could rename this to be metamask specific.  Not a huge deal though